horreum-web/package.json

```
@@ -4,9 +4,9 @@
   "private": true,
   "dependencies": {
     "@monaco-editor/react": "4.6.0",
-    "@patternfly/patternfly": "4.224.4",
-    "@patternfly/react-core": "4.277.0",
-    "@patternfly/react-table": "4.113.5",
+    "@patternfly/patternfly": "^5.1.0",
+    "@patternfly/react-core": "^5.1.1",
+    "@patternfly/react-table": "^5.1.1",
```

```suggestion
    "@patternfly/patternfly": "5.1.0",
    "@patternfly/react-core": "5.1.1",
    "@patternfly/react-table": "5.1.1",
```
please, use specific versions

---

This is for a UI library, is there an issue with putting the version range in package.json and relying on package-lock to preserve the specific version?

---

I made the change and see `"vite-tsconfig-paths": "^4.2.3"` in `master` is that also being updated to follow a specific version?
https://github.com/Hyperfoil/Horreum/blob/master/horreum-web/package.json#L49

---

it should be yes

---

> This is for a UI library, is there an issue with putting the version range in package.json and relying on package-lock to preserve the specific version?

Until we have unit tests for the UI, this limits the probability of in compatible transient deps breaking things. It isn't ideal, but neither is having a lack of UI testing. We can revisit this once we have UI tests that will detect breaking changes

---

@willr3 make sure your lock file is up to date with your current change